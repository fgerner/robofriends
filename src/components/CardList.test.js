import {shallow} from 'enzyme';
import CardList from "./CardList";

describe('Cardlist component', () => {
    it('should render without crashing', function () {
        const mockRobots = [
            {
                id: 1,
                name: 'test',
                username: 'test',
                email: 'test@test.com'
            }
        ]
        expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
    });
})