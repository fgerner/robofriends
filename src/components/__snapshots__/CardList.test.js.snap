// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cardlist component should render without crashing 1`] = `ShallowWrapper {}`;
